docs/core/entity/camera.md

```diff
@@ -93,15 +93,37 @@ A common way for a camera entity to render a camera still image is to pass the s
 
 ### WebRTC streams
 
-WebRTC enabled cameras can be used by facilitating a direct connection with the home assistant frontend. This usage requires `CameraEntityFeature.STREAM` with `frontend_stream_type` set to `StreamType.WEB_RTC`. The integration should implement `async_handle_web_rtc_offer` which passes the frontend's SDP offer to the device and returns back the answer.
+WebRTC enabled cameras can be used by facilitating a direct connection with the home assistant frontend. This usage requires `CameraEntityFeature.STREAM` with `frontend_stream_type` set to `StreamType.WEB_RTC`.
+
+The integration should implement the following methods:
```

Comment:
We should clarify when integrations need to implement these methods. As there are already defaults in place, normally integrations that want to add a camera platform don't need to implement them, right?

Reply:
To provide native support, they must implement the given methods. If the integration supports only HLS, than they don't need to implement it